### DISCUSSION TITLE
npm create electron-vite 创建的新项目调试断点无效

### DISCUSSION THREAD

**Discussion by yedielnthabit** [electron-vite/electron-vite-vue#401] [Q&A] [answered]
用npm create electron-vite创建的新项目在 main.ts的win = new BrowserWindow处下断点无效程序不会暂停，但是克隆仓库下来的却是可以的 尝试复制.vscode里面的文件到npm create electron-vite创建的项目中仍然不行

**Comment by caoxiemeihao** [comment]
需要完整的配置，你仔细对比下 vite.config.js 尤其是 sourcemap

**Reply by yedielnthabit** [reply]
也就是说在npm create electron-vite 之后还需要拷贝一份源码当中的vite.config.js吗 初始的vite.config.js确实少了非常多内容

**Reply by caoxiemeihao** [reply, marked as answer]
`npm create electron-vite` 不可能提供非常过的功能，你可以理解它只是 `npm create vite`  加了一小部分 Electron 必要代码而已，与 Vite 提供的基础模板保持一致

**Reply by yedielnthabit** [reply]
明白了，理解这个逻辑就知道该怎么弄了，感谢回答
